### ISSUE TITLE
Accessibility in XYChart of amcharts

### ISSUE THREAD

**Issue by soumyaa1804** [amcharts/amcharts4#4105]
**Question**

I want to setup accessibility in my line and bar charts of amcharts in my project. I am using Voice over (mac tool) to test it. I want that on focus it should announce the x value and y value. I am not using bullets on line chart but only tooltip.

I tried adding the props given on this page: https://www.amcharts.com/docs/v4/concepts/accessibility/

But as we don't have bullets so it skipped all data points. Can someone please help how to achieve it?

**Environment (if applicable)**

amCharts version: v4
Browser: Chrome
React application

**Additional context**

None

**Comment by martynasma** [comment]
You will need to add bullets so there's a focusable item.

You can make them invisible (e.g. by having `fillOpacity` set to 0), but they need to be there.
